src/bitmap.c

```diff
@@ -0,0 +1,189 @@
+/*	bitmap.c
+ *
+ *	Source: https://msdn.microsoft.com/de-de/library/windows/desktop/dd145119(v=vs.85).aspx
+ *
+ *	Usage:
+ *		HBITMAP hBitmap;
+ *		HDC hdc;
+ *		PBITMAPINFO pBitmapInfo = createBitmapInfoStruct(hBitmap);
+ *		createBmpFile("picture.bmp"), pBitmapInfo, hBitmap, hdc);
+ */
+
+#if defined(WIN32_LEAN_AND_MEAN)
```

Comment:
Both should define with with `-mwin32`, although I have no idea what that'll do to the rest of the build.

Since we're not targeting Windows installs without either Cygwin or MSYS2, just `defined(__CYGWIN__) || defined(__MSYS__)` should be sufficient.